src/App.js

```diff
@@ -0,0 +1,46 @@
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { ToastContainer } from 'react-toastify';
+import LoginPage from './pages/LoginPage';
+import ProtectedRoute from './pages/ProtectedRoute';
+import './App.css';
+import 'react-toastify/dist/ReactToastify.css';
+import Vehicles from './pages/Vehicles';
+import ReserveForm from './pages/ReserveForm';
+import Reservations from './pages/Reservations';
+import AddVehicle from './pages/AddVehicle';
+import RemoveVehicle from './pages/RemoveVehicle';
+import VehicleDetails from './pages/VehicleDetails';
+import VehicleReserve from './pages/VehicleReserve';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/HERMON-1995/book_vehicle_rental-frontend/assets/57421823/f0b26a17-c3b4-40df-9781-efdc0fb08a5b)
 

That way, your code will meet the latest standards. 💪